com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/MotionVectorVertexShaderCommon.hlsl

```diff
@@ -85,7 +85,11 @@ void MotionVectorPositionZBias(VaryingsToPS input)
 #endif
 }
 
+#ifdef HAVE_VFX_MODIFICATION
+PackedVaryingsType MotionVectorVS(inout VaryingsType varyingsType, AttributesMesh inputMesh, AttributesPass inputPass, AttributesElement inputElement)
```

Comment:
just to be sure we avoid error in case of future change, it is better to do it like this:

PackedVaryingsType MotionVectorVS(inout VaryingsType varyingsType, AttributesMesh inputMesh, AttributesPass inputPass, 
#ifdef HAVE_VFX_MODIFICATION
, AttributesElement inputElement
#endif
)

I know it could looks like a matter of taste, but often, as the other part of the code isn't compile, we forget to update both side.